src/openforms/analytics_tools/utils.py

```diff
@@ -130,11 +125,13 @@ def update_analytical_cookies(
         Cookie.objects.filter(name__in=[cookie["name"] for cookie in cookies]).delete()
 
 
-def update_csp(csps: list[CSPDict], create: bool):
+def update_csp(
+    config_model: "AnalyticsToolsConfiguration",
```

Comment:
it's an instance, not a model, so the variable name can be improved upon

"a django model" refers the model class and associated table :)